packages/palette/src/elements/TextArea/__tests__/TextArea.test.tsx

```diff
@@ -87,14 +87,6 @@ describe("TextArea", () => {
     })
   })
 
-  it("takes a default value", () => {
-    getWrapper({ defaultValue: "This is the default value" })
-    expect(onChange).toHaveBeenCalledWith({
-      value: "This is the default value",
-      exceedsCharacterLimit: false,
-    })
-  })
```

Comment:
Removing this spec since `onChange` shouldn't be called on mount. 🤞 this doesn't break something but I'll handle it if it does.